tests/integration/dynamic_ip_test.py

```diff
@@ -576,6 +606,71 @@ def test_ipv6_autoconf_only(dhcpcli_up):
     libnmstate.apply(desired_state)
 
 
+@pytest.mark.xfail(reason='DHCP server does not run behind vlan', strict=True)
+def test_linux_bridge_over_vlan_over_bond_over_slave_single_transaction(
```

Comment:
The name does not express that there is no dhcp server, which is the main point of this test.